packages/regl-worldview/rollup.config.js

```diff
@@ -45,6 +47,7 @@ export default [
       babel(getBabelOptions({ useESModules: true })),
       commonjs({ include: "node_modules/**" }),
       replace({ "process.env.NODE_ENV": JSON.stringify("development") }),
+      uglify(),
```

Comment:
Might want to consider using `rollup-plugin-terser` instead of uglify if we want `for await` support (see https://github.com/cruise-automation/webviz/blob/530a08ef98ea43d64b85982c26e6d1b81e23bdb5/webpack.config.js#L84). Not sure if that still applies though (and not sure if we even have that in this codebase, but we might have soon if we add more webviz stuff here).